include/cop1.h

```diff
@@ -34,6 +34,8 @@
 #define C1_CAUSE_INVALID_OP         0x00010000
 #define C1_CAUSE_NOT_IMPLEMENTED    0x00020000
 
+#define C1_FS_BIT                   (1<<24)
```

Comment:
I think we should name this `C1_FCR31_FS`. I think we usually put the register name first for which the mask must be used.